ironmq/src/main/scala/akka/stream/alpakka/ironmq/IronMqClient.scala

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ironmq
```

Comment:
We use the convention of `scaladsl` and `javadsl` packages for the Scala and Java APIs